assignments/arrays.js

```diff
@@ -63,7 +63,12 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for(let i = 0; i === inventory["id" : "33"; i++){
+
+}
+
+console.log(`Car 33 is a (inventory*car year goes here* *car make goes here* *car model goes here*` );
 
 
 
```

Comment:
Challenge 2, It wants you to select the last car of the array, You can hard code it by writing inventory[49], but given the if in the future they add more cars into the inventory, then it won't be the last one anymore. So we know by counting the object inside the array, it is 50. But we know computers count by based 0, so think of the "difference" between the two and put that logic into inventory[] so it can select the last one every time.